geoportailv3/externs/localforage.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @fileoverview Externs for localForage
+ *
+ * @see https://github.com/mozilla/localForage
+ * @externs
+ */
+
+/**
+ * @typedef {{
+ *   description: (?string|undefined),
+ *   driver: (Array<*>|undefined),
+ *   name: (?string|undefined),
+ *   size: (?number|undefined),
+ *   storeName: (?string|undefined),
+ *   version: (?*|undefined)
+ * }}
+ */
+var LocalForageConfig;
+
+/**
+ * @constructor
+ * @param {LocalForageConfig=} options
+ */
+function Localforage(options) {}
+
+/**
+ * @param {LocalForageConfig} options
+ */
+Localforage.prototype.config = function(options) {};
+
+/**
+ * @param {string} key
+ * @param {*} value
+ * @return {Promise} 
+ */
+Localforage.prototype.setItem = function(key, value) {};
+
+/**
+ * @param {string} key
+ * @return {Promise<*>}
+ */
+Localforage.prototype.getItem = function(key) {};
+
+/**
+ * @param {string} key
+ * @return {Promise}
+ */
+Localforage.prototype.removeItem = function(key) {};
+
+/**
+ * @return {Promise}
+ */
+Localforage.prototype.clear = function() {};
+
+/**
+ * @return {Promise<Array<string>>}
+ */
+Localforage.prototype.keys = function() {};
+
+/**
+ * @return {Promise<number>}
+ */
+Localforage.prototype.length = function() {};
```

Comment:
The same, why it doens't use the one from ngeo

Reply:
Well, there is no such extern in ngeo.
We will remove it when switching to ES6 and building with webpack.

Reply:
ok